.github/actions/deploy_v2/action.yml

```diff
@@ -49,7 +49,7 @@ runs:
           echo "DEPLOY_REF=${{ github.ref }}" >> $GITHUB_ENV
           hostname=$(jq -r '.additional_hostnames[0]' ${tf_vars_file})
           if [ ${app_environment} == "production" ]; then
-            echo "deploy_url=https://api2.publish-teacher-training-courses.service.gov.uk" >> $GITHUB_OUTPUT
+            echo "deploy_url=https://api3.publish-teacher-training-courses.service.gov.uk" >> $GITHUB_OUTPUT
```

Comment:
Can you clarify why api3, please?

Reply:
I tried www2 (and api2 for the API) but because of [this](https://github.com/DFE-Digital/publish-teacher-training/blob/0686327a0d76956c0d157b36a8021987544a7452/config/routes.rb#L13-L15) it wouldn't work and kept redirecting so I changed it all to use www2 and api3 to get it working and for consistency.